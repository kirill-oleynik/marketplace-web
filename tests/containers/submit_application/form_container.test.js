const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const SubmitApplicationFormContainer = require('../../../app/containers/submit_application/form_container').default;

const fakeStore = (state = {}) => ({
  subscribe() {},
  dispatch() {},
  getState() { return state; }
});

const submitApplication = {
  errors: {}
}

describe('#render', () => {
  test('it renders correctly', () => {
    const store = fakeStore({
      submitApplication
    });

    const component = shallow(
      <SubmitApplicationFormContainer
        t={(translate) => translate}
        submitApplication={() => {}}
        toggleSubmitApplicationModal={() => {}}
      />,
      { context: { store } }
    )

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
