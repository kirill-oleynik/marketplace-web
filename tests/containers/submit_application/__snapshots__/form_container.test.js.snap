// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Translate(SubmitApplicationFormContainer)
  errors={Object {}}
  submitApplication={[Function]}
  t={[Function]}
  toggleSubmitApplicationModal={[Function]}
/>
`;
